Modules/Relationships/src/Relationships.Domain/Aggregates/RelationshipTemplates/RelationshipTemplate.cs

```diff
@@ -69,6 +69,15 @@ public void AllocateFor(IdentityAddress identity, DeviceId device)
         Allocations.Add(new RelationshipTemplateAllocation(Id, identity, device));
     }
 
+    public void AnonymizeForIdentity(string didDomainName)
+    {
+        EnsureIsPersonalized();
```

Comment:
I don't think that an un-personalized template has to throw here. We should just ignore it.

Reply:
So it should just return silently?